Makefile

```diff
@@ -81,7 +87,7 @@ generate: controller-gen conversion-gen
 docker-build: test docker-build-ci
 
 # Build the docker images
-docker-build-ci:
+docker-build-ci: manifests
```

Comment:
Should we be committing the generated files like we do with the rest of the generated code so we do not need to regenerate manifests on every build?

Reply:
initially it was because I wanted the build version information from the CI run to get injected when we built the manifests. but after switching over to using kustomize, im not sure it's needed anymore. I'll double check and drop it if we don't need to